wgpu-types/src/features.rs

```diff
@@ -1301,6 +1304,19 @@ bitflags_array! {
         /// This is a web and native feature.
         const TEXTURE_COMPRESSION_ASTC = WEBGPU_FEATURE_TEXTURE_COMPRESSION_ASTC;
 
+
+        /// Allows the 3d dimension for textures with ASTC compressed formats.
+        ///
+        /// This feature must be used in combination with TEXTURE_COMPRESSION_ASTC to enable 3D textures with ASTC compression.
+        /// It does not enable the ASTC formats by itself.
+        ///
+        /// Supported Platforms:
+        /// - Desktop (some)
+        /// - Mobile (some)
```

Comment:
this isn't terrible helpful imho, how about

```suggestion
        /// Supported Platforms:
        /// - Vulkan (some)
        /// - Metal on Apple3+
        /// - OpenGL/WebGL (some)
        ///
        /// Not Supported:
        /// - DX12
```

Reply:
@Wumpf much better, thank you!